@gorkem-bwl  build check is still failing

```
Error: src/Components/layout/AppLayout.tsx(5,1): error TS6133: 'BackgroundSVG' is declared but its value is never read.
Error: src/Components/layout/AppLayout.tsx(16,8): error TS6133: 'mode' is declared but its value is never read.
Error: Process completed with exit code 2.
```

--- a/client/src/Components/layout/AppLayout.tsx
+++ b/client/src/Components/layout/AppLayout.tsx
@@ -1,9 +1,6 @@
 import Box from "@mui/material/Box";
 import { useState, useEffect, useRef } from "react";
 import { useTheme } from "@mui/material/styles";
-import { useSelector } from "react-redux";
-import BackgroundSVG from "@/assets/Images/background.svg";
-import type { RootState } from "@/Types/state";
 import { OfflineBanner } from "@/Components/design-elements";
 import { setServerUnreachableCallback, get } from "@/Utils/ApiClient";
 
@@ -13,7 +10,6 @@ interface AppLayoutProps {
 
 const AppLayout = ({ children }: AppLayoutProps) => {
 	const theme = useTheme();
-	const mode = useSelector((state: RootState) => state.ui.mode);
 	const [serverUnreachable, setServerUnreachable] = useState(false);
 	const retryIntervalRef = useRef<number | null>(null);
 

